requirements.txt

```diff
@@ -5,3 +5,4 @@
 
 Adafruit-Blinka
 adafruit-circuitpython-hid
+sphinx>=4.0.0
```

Comment:
This needs to go into a new `/docs/requirements.txt` file, not in the `/requirements.txt` file. Please create the new file, add this to it, and remove it from here.